esmvalcore/config/_config.py

```diff
@@ -0,0 +1,130 @@
+"""Functions dealing with config-user.yml / config-developer.yml."""
+import collections.abc
+import fnmatch
+import logging
+import os
+import sys
+from functools import lru_cache
+from pathlib import Path
+
+import yaml
+
+from esmvalcore.cmor.table import CMOR_TABLES, read_cmor_tables
+from esmvalcore.exceptions import RecipeError
+
+logger = logging.getLogger(__name__)
+
+TASKSEP = os.sep
+
+CFG = {}
+
+if sys.version_info[:2] >= (3, 9):
+    # pylint: disable=no-name-in-module
+    from importlib.resources import files as importlib_files
+else:
+    from importlib_resources import files as importlib_files
```

Comment:
Note to reviewer: this can only be properly tested by using a different Python version than the latest. This is done by the GitHub actions, but not on CircleCI where we run the tests that collect coverage information that is sent to Codecov.

Reply:
It could probably be done, but I'm not sure it's worth the effort. We do have the nightly GitHub action tests to test this with Python 3.8, so we would notice if it breaks.